.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-    python: python3.12
+    python: python3.10
```

Comment:
This should probably be bumped down too our minimum supported version, which is 3.9 currently.

Reply:
Yup [looks like](https://github.com/pre-commit/pre-commit/blob/504149d2ca57f3115ccd4fa9c6ae692929b2282a/.pre-commit-config.yaml) it's not needed so we can remove it.